### ISSUE TITLE
Basic auth and HTTPS

### ISSUE THREAD

**Issue by alustrement** [Microsoft/tfs-cli#63]
Using basic auth with an HTTPS endpoint cause the following error :
```
tfx login --auth-type basic --service-url https://server/tfs --username user --password pwd
Connection failed. Check your internet connection & collection URL.
Message: unable to get local issuer certificate
```
It seems to be the same problem I had with Git : http://blogs.msdn.com/b/phkelley/archive/2014/01/20/adding-a-corporate-or-self-signed-certificate-authority-to-git-exe-s-store.aspx
So how I can specify my certificate file ?

(Trace doesn't help, CLI version is 0.3.3)

**Comment by markjgardner** [comment]
Same problem.  Alternatively, I would accept an option to ignore certificate validation failures.

**Comment by trevorsg** [comment]
Thanks. I will look into this, but it won't be until after New Years.

**Comment by grawcho** [comment]
I would like to add that it will be nice to have the ability to use personal access token in on premise TFS server. this issue is blocking my team selecting TFS as our cross platform automation system.

**Comment by brettjacobson** [comment]
Any update?  This still appears to be a problem.  And asking users to send their enterprise domain credentials across the network unencrypted when using on premises TFS seems like a rather large security hole that Microsoft has not addressed and the problem has been known for over 6 months.

**Comment by bryanmacfarlane** [comment]
@grawcho - note that this tfs-cli issue and "cross platform automation system" are a bit orthogonal.  The new vsts-agent supports ntlm across platforms so you should be unblocked on the automation system side: https://github.com/Microsoft/vsts-agent
